conans/client/conf/__init__.py

```diff
@@ -147,7 +149,7 @@
 class ConanClientConfigParser(ConfigParser, object):
 
     def __init__(self, filename):
-        ConfigParser.__init__(self)
+        ConfigParser.__init__(self, allow_no_value=True)
```

Comment:
allow_no_value not used, right?

Reply:
I think it is actually needed for ``conan config set plugins.my_plugin`` as the normal behaviour would be ``conan config set plugins.my_plugin=plugin_value_if_something``

Reply:
Yes, that parameter is needed